app/src/main/java/hr/dice/filipbionda/tmdbpractice/MainActivity.kt

```diff
@@ -0,0 +1,20 @@
+package hr.dice.filipbionda.tmdbpractice
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import hr.dice.filipbionda.tmdbpractice.ui.theme.TMDBPracticeTheme
+import hr.dice.filipbionda.tmdbpractice.ui.welcomescreen.WelcomeScreen
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
```

Comment:
Lets not have system bars transparent for the sake of this practice. It wasn't outlined in Figma

Reply:
done